pkg/visor/init.go

```diff
@@ -105,20 +116,27 @@ func registerModules(logger *logging.MasterLogger) {
 	ebc = maker("event_broadcaster", initEventBroadcaster)
 	ar = maker("address_resolver", initAddressResolver)
 	disc = maker("discovery", initDiscovery)
+	tr = maker("transport", initTransport, &ar, &ebc)
+
+	sc = maker("stun_client", initStunClient)
+	sudph = maker("sudph", initSudphClient, &sc, &tr)
+	sctpr = maker("sctpr", initSctprClient, &tr)
```

Comment:
`stcpr` as in Skywire TCP with automatic address Resolution. Same for `stcp`.

Reply:
My bad. Will make the changes.